### PR TITLE
[yang] add Yang model for XCVRD_LOG|Y_CABLE

### PR DESCRIPTION
**Pull request by vdahiya12**
#### Why I did it

Adding yang model for CONFIG_DB table XCVRD_LOG|Y_CABLE.
Introduced by 
https://github.com/sonic-net/sonic-utilities/blob/master/config/muxcable.py#L1230-L1235

sign-off: Vaibhav Dahiya [vdahiya@microsoft.com]
##### Work item tracking
- Microsoft ADO **(number only)**:
- 30044479

#### How I did it
Added the changes in sonic-yang-models
#### How to verify it
UT test 

```
==================================================================================== test session starts ====================================================================================
platform linux -- Python 3.9.2, pytest-6.0.2, py-1.10.0, pluggy-0.13.0
rootdir: /sonic/src/sonic-yang-models
plugins: pyfakefs-5.2.3, cov-2.10.1
collected 3 items                                                                                                                                                                           

tests/test_sonic_yang_models.py ..                                                                                                                                                    [ 66%]
tests/yang_model_tests/test_yang_model.py .                                                                                                                                           [100%]

===================================================================================== 3 passed in 2.06s =====================================================================================
```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Review by prgeor — APPROVED** [review]

**Comment by ganglyu** [comment]
sonic-yang-mgmt unit test is broken

File: src/sonic-yang-models/tests/files/sample_config_db.json [hunk 1]
```diff
@@ -2723,6 +2723,12 @@
             }
         }
     },
+
+    "XCVRD_LOG": {
+        "Y_CABLE": {
+            "log_verbosity": "notice"
+        }
+    },
```
Comment (ganglyu) [inline src/sonic-yang-models/tests/files/sample_config_db.json:2731]: This table should be at the same level as BANNER_MESSAGE.

Comment (vdahiya12) [reply src/sonic-yang-models/tests/files/sample_config_db.json]: fixed

**Review by wen587 — APPROVED** [review]
lgtm

File: src/sonic-yang-models/tests/files/sample_config_db.json [hunk 2]
```diff
@@ -2723,6 +2728,7 @@
             }
         }
     },
+
```
Comment (wen587) [inline src/sonic-yang-models/tests/files/sample_config_db.json:2731]: extra space

Comment (vdahiya12) [reply src/sonic-yang-models/tests/files/sample_config_db.json]: removed

**Review by ganglyu — APPROVED** [review]

**Review by wen587 — APPROVED** [review]

**Comment by vdahiya12** [comment]
@yxieca please help merge

**Comment by wen587** [comment]
@qiluo-msft , please help merge

**Review by qiluo-msft — APPROVED** [review]

**Comment by wen587** [comment]
Hi @vdahiya12 , please help contact branch owner and backport to previous branch

**Pull request merged**